### ISSUE TITLE
wite-up a real README.md

### ISSUE THREAD

**Issue by boromil** [SlashDB/angular-slashdb#5]
the current one is only a placeholder
would be nice to have one
we can model it on other angular plugins

**Comment by boromil** [comment]
I've added an initial commit to the README.md

**Comment by agilevic** [comment]
Let's try these to improve the README.

Opening statement:

[SlashDB](http://slashdb.com/) automatically creates REST APIs on top of traditional databases for reading and writing by authorized applications without the need of SQL queries. Angular-slashdb (this project) is a small plug-in, allowing you to use SlashDB features more easily in your AngularJS app. Together they allow developers to avoid tedious work and focus on application features that matter.

Next:

Code snippet showing typical usage for GET and "see examples folder for more details."

Then, Table of Contents.

Lastly, the rest of your doc in the order as you have it.

**Comment by agilevic** [comment]
Move this part under Installing header

This plug-in depends on:

SlashDB >= 0.9.7
AngularJS >= 1.5.7 and < 2.0

**Comment by boromil** [comment]
I've added "Table of contents"  and did the changes that @agilevic requested.
The _GET_ example is [here](https://github.com/SlashDB/angular-slashdb#fast-example).
Also made going around the README a bit easier with _Back to top_ :)

**Comment by boromil** [comment]
btw. small errors, (leftover coma or something like that) can be easily corrected directly in github by selecting the README.md file and editing it :)

**Comment by agilevic** [comment]
1. Please add a 1 line description what each function does in plain English.
2. Make all examples based on the Chinook database and verify they actually would work.

**Comment by boromil** [comment]
1. Done
2. Even if it would be based on Chinook they won't work - you need a correctly configured angular app, permissions on the endpoint side, SQL query defined (i.e. for _getQueries_ and _executeQuery_) - can't really predict in what context those examples will be use.
As a user of this plugin I would treat them as templates to use in my app - rather than real code, that's why thy are so generic.

**Comment by agilevic** [comment]
Good points. Once SlashDB is up on beta.slashdb.com, lets upgrade the example app to showcase how to use it correctly. We will make sure that the service fits the example app so that developer's first experience is easy.
